src/List.test.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+
+import List from './List';
+
+describe('List', () => {
+  const handleClick = jest.fn();
+
+  function renderList({ tasks, handleClickDelete }) {
+    return render((
+      <List
+        tasks={tasks}
+        handleClickDelete={handleClickDelete}
+      />
+    ));
+  }
+
+  context('task가 없을 경우', () => {
+    const tasks = [];
+
+    it('"할 일이 없어요!"를 표시한다.', () => {
+      const { container } = renderList({ tasks, handleClick });
```

Comment:
```suggestion
      const { container } = renderList({ tasks, handleClickDelete: handleClick });
```
맵핑되는 함수명이 달라서 에러가 발생하는 것 같아요. 지금과 같이 함수명을 다른 상황에서 객체로 넘기는 경우엔 이렇게 맵핑 해줘야할 것 같네요.

Reply:
구조분해할당에 대해서 봐보시면 도움 될 것 같아요~

- https://ko.javascript.info/destructuring-assignment#ref-1545